### ISSUE TITLE
Modify reply text field so that it will auto-capitalise correctly

### ISSUE THREAD

**Issue by ineffyble** [mastodon/mastodon#20618]
### Discussed in https://github.com/mastodon/mastodon/discussions/20590

<div type='discussions-op-text'>

<sup>Originally posted by **DavidToddMcCarty** November 14, 2022</sup>
I was wondering if there was any way to default to a reply, to begin with a capital letter just like it does with a new post. Seems like an easy fix, no?</div>

As mentioned in replies: Twitter and Facebook have an empty text field, but Mastodon starts with a mention, which I guess is being detected as the start of a sentence by whichever keyboard you are using (Apple, Google, etc)..

**Comment by trwnh** [comment]
#18708

This is related to the (controversial) decision that Twitter made to split its mentions out to separate addressing metadata

**Comment by ineffyble** [comment]
True, it's not a requirement though. There's solutions other than going the same route.

**Comment by Gargron** [comment]
I'm not aware of a way to make the browser capitalize input from the middle.
